articles/videos/get-started/05_01-authorize-id-tokens-access-control.md

```diff
@@ -0,0 +1,139 @@
+---
+description: What an ID Token is and how you can add custom claims to make access control decisions for your users.
+classes: video-page
+---
+# Authorize: ID Tokens and Access Control
+
+Learn about using Auth0 authorization via the use of ID Tokens and understand what an ID Token is and how to add custom claims to customize them and make access control decisions for your users.
```

Comment:
A lot packed into this sentence. Maybe "Learn about Auth0 authorization via the use of ID Tokens. Understand what an ID Token is, and learn how to add custom claims to customize tokens and make access control decisions for your users."

Reply:
Much better